screeps-profiler.js

```diff
@@ -14,20 +14,36 @@ function setupProfiler() {
     profile(duration, filter) {
       setupMemory('profile', duration || 100, filter);
     },
+    background(duration, filter) {
+      setupMemory('background', duration || false, filter);
+    },
+    clear() {
```

Comment:
I don't like that this behaves differently in different types of profiling.  It could conceivably work the same way for all profile types, all it needs to do is pass all current values back to `setupMemory` after calculating a new duration `disableTick - enabledTick`.

Also, I think that `restart` may be a better name.